R/ResampleDesc.R

```diff
@@ -45,8 +46,15 @@
 #'     \dQuote{Subsample} between 0 and 1. Default is 2 / 3.}
 #'   \item{reps [\code{integer(1)}]}{Repeats for \dQuote{RepCV}. Here \code{iters = folds * reps}.
 #'     Default is 10.}
-#'   \item{folds [\code{integer(1)]}}{Folds in the repeated CV for \code{RepCV}.
+#'   \item{folds [\code{integer(1)}]}{Folds in the repeated CV for \code{RepCV}.
 #'     Here \code{iters = folds * reps}. Default is 10.}
+#'   \item{horizon [\code{integer(1)}]}{Number of observations to forecast for \code{GrowthCV}
+#'    and \code{FixedCV}.}
+#'   \item{initial.window [\code{integer(1)}]}{Initial number of observations to start with
+#'    in \code{GrowthCV} and \code{FixedCV}.}
+#'   \item{size [\code{integer(1)}]}{Number of observations in data set. Used for \code{GrowthCV}
+#'    and \code{FixedCV}.}
```

Comment:
Imho size shouldn't even be an optional argument; if the user wants to see the number of iterations, he can tentatively create a `ResampleInstance` and look at it's print output.